### ISSUE TITLE
Check data collected files

### ISSUE THREAD

**Issue by aremazeilles** [AdrianaTorres/Irregular_Terrains#17]
To complete the picture, it would be good to check the collected files by the testbed, and see how they are named. This is to make sure that the PI Manager can then automatically call your program.

Could you give us the structure (file naming) of the information that is to be collected in an experiment?

**Comment by AdrianaTorres** [comment]
Do you mean the  inputs of the PIs or the files we then convert into the inputs?

**Comment by aremazeilles** [comment]
Well, both :)

**Comment by AdrianaTorres** [comment]
Ok. 
- The name of the files that are then converted into the input files is: (metric_of_interest)_(name_of_Vicon_capture).csv/yaml
- The name of the input files is: (metric_of_interest)_(exo/noExo in case of gait parameters).csv/yaml.
Actually this naming is not consistent at all, because I tried to name them following the EUROBENCH Data Format and added the needed information for me to know to from which capture/condition do they come from. We can rename them following what we saw in one of the latest meetings so that everything is better defined and consistent.

**Comment by aremazeilles** [comment]
* Are you repeating the experiment? 
* Do you do so changing the configuration of the irregular terrain (where do you encode the structure of the irregular terrain?)

I'll comment on this later on, I foresee some issues  with the default eurobench-way, as some metric may combine data from different conditions (if we consider the exo/no exo as a condition parameter).

**Comment by AdrianaTorres** [comment]
Yes, I am doing 4 runs per terrain condition (14). A total of 56 runs. We can discuss the naming in a meeting and talk about the problems we may have if you prefer.

**Comment by aremazeilles** [comment]
In the file stored as reference, I can see the use of `_plano1` for com, feettrajectorie and gaitEvents, and the use of `_exo, _noExo` for the `GaitParameters`. 

is `_plano1` related to hte config of the testbed?
I would expect to see the tag `_exo`, `_noExo` in all your files: you listed 56 runs, but I would assume it is 56 runs without the exo, and 56 runs with the exo, am I correct?

**Comment by aremazeilles** [comment]
To complete the integration, I think it would be to chat directly looking at the date type and agree on a naming for all the files you have. Would it be ok to set a meeting on this?
